infrastructure/backend_ecr.tf

```diff
@@ -0,0 +1,41 @@
+locals {
+  backend_image_uri = "${data.aws_caller_identity.current.account_id}.dkr.ecr.eu-central-1.amazonaws.com/${aws_ecr_repository.backend_image.name}@${data.aws_ecr_image.backend_image.image_digest}"
+}
+
+resource "aws_ecr_repository" "backend_image" {
+  name                 = "${terraform.workspace}-pycon-backend"
+  image_tag_mutability = "MUTABLE"
+
+  image_scanning_configuration {
+    scan_on_push = false
+  }
+}
+
+resource "aws_ecr_lifecycle_policy" "backend_image_cleanup" {
+  repository = aws_ecr_repository.backend_image.name
+
+  policy = <<EOF
+{
+    "rules": [
+        {
+            "rulePriority": 1,
+            "description": "Expire images older than 1 days",
+            "selection": {
+                "tagStatus": "untagged",
+                "countType": "sinceImagePushed",
+                "countUnit": "days",
+                "countNumber": 1
```

Comment:
is this something we want?

Reply:
Hmm I went for storing only the latest image, and delete anything older as we do a deploy every time we push to master so having all those docker images that we do not use seems a bit wasteful

We also need to improve our deployment to build/release the BE only when we change something there

Reply:
so it doesn't delete the last one?

Reply:
yup only untagged images, so when we release a new build the new image gets "latest" and the previous one untagged